mne/datasets/hf_sef/hf_sef.py

```diff
@@ -98,3 +95,9 @@ def data_path(dataset='evoked', path=None, force_update=False,
 
     _do_path_update(path, update_path, key, name)
     return destdir
+
+###############################################################################
+# References
+# ----------
+#
+# .. footbibliography::
```

Comment:
I think this needs to be removed and should be reflected in lines 45-48

Reply:
actually I think this is correct.  We want the references moved to the bottom of the example (previously they were at the top, which is non-standard)

Reply:
Maybe I don't get it ... is this file an example? Looks like a normal .py module file to me

Reply:
Indeed this is a standard module file so I think it should be moved up to the docstring

Reply:
ah, oops.  I didn't look closely enough.